install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -804,21 +804,19 @@ ciliumEndpointSlice:
   # limit: The sustained request rate in requests per second. The maximum rate that can be configured is 50.
   # burst: The burst request rate in requests per second. The maximum burst that can be configured is 100.
   rateLimits:
-  - nodes: 0
-    limit: 10
-    burst: 20
-  - nodes: 100
-    limit: 50
-    burst: 100
-
+    - nodes: 0
```

Comment:
i wonder if the linter or the make check fixed these? I didnt manually change these.

Reply:
fixed.